src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -190,9 +141,16 @@ class ValidationStep extends React.Component {
                     </View>
                 )}
                 {!maxAttemptsReached && state === BankAccount.STATE.PENDING && (
-                    <ReimbursementAccountForm
-                        reimbursementAccount={this.props.reimbursementAccount}
+                    <Form
+                        formID={ONYXKEYS.FORMS.VALIDATION_STEP_FORM}
+                        submitButtonText={
```

Comment:
AFAIK the button text here is always `'validationStep.buttonText'`, so no need for this logic